docs/getting-started/aws-remote-access.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Remote Access for Apache Gravitino on AWS"
+slug: /getting-started/aws-remote-access
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Accessing Apache Gravitino on AWS externally
+
+When you deploy Gravitino on AWS, accessing it externally requires
+some additional configuration due to how AWS networking works.
```

Comment:
Please don't break sentences over multiple lines as this breaks some tooling and can introduce errors into the content.

Reply:
I hear you, Justin.
May I know what tools I am breaking?
I believe I have articulated beforethe necessity to break long lines at punctuations or sentence ends. Long lines make our lives difficult. Difficult to read the source markdown, difficult to review changes on a "per-line" basis.

Reply:
This breaks grammar checkers like Grammarly and injects false warnings, making these tools more difficult to use. GitHub shows word-by-word changes, so there is no need to break sentences over lines. I don't know if you are using a grammar checker, but as you can see, there are several issues here that have been introduced by breaking sentences over lines.

Reply:
This also means that reviewers have to do more work (i.e remove the line breaks) to review these changes as errors are hidden in GitHub or their IDE.

Reply:
Why is that necessary?

Reply:
> there are several issues here that have been introduced by breaking sentences over lines.

Really? How many syntax errors are easier to spot if we don't have line breaks?

Reply:
> A grammar checker can see and visually display the issues right away in the GitHub interface or in an IDE.

Can you tell me where is the guideline on using grammar checkers?
I'm always using vim (as my IDE). Is this line-wrapping thing related to IDE?

> To see any problems like this, a reviewer must manually remove the new lines.

Give me an example please? Why we have to remove this new lines before review?
Why we cannot just treat them as a book?

> This means it's more work to review these changes, and as you can see, it introduces errors that require more work to fix.

If we have to remove new lines to review, then I tend to agree to remove these new lines.